lib/apparmortest.pm

```diff
@@ -117,6 +121,11 @@ sub aa_status_stdout_check {
     assert_script_run("aa-status | head -$lines | tail -$total_line | sed 's/[ \t]*//g' | grep -x $profile_name");
 }
 
+sub ip_fetch {
+    my $ip = script_output("hostname -I | cut -d ' ' -f1");
+    return $ip;
```

Comment:
Maybe the code should deal with error when hostname return fail.

Reply:
I know what you concern. Thanks.
In this situation If the $ip variable gets NULL then the following scenario will fail.
So I will keep the code unchanged.